jwql/website/apps/jwql/oauth.py

```diff
@@ -259,7 +260,7 @@ def login(request, user):
     # Redirect to oauth login
     global PREV_PAGE
     PREV_PAGE = request.META.get('HTTP_REFERER')
-    redirect_uri = os.path.join(get_base_url(), 'authorize')
+    redirect_uri = f"{get_base_url()}{reverse('jwql:authorize')}"
```

Comment:
maybe a mix of the two would actually be a bit better here...

```suggestion
    redirect_uri = os.path.join(get_base_url(), reverse('jwql:authorize'))
```